front/src/common/ReleaseInformations/LicenseAttributions.tsx

```diff
@@ -11,6 +11,18 @@ type AttributionLicense = {
   license: string;
 };
 
+// Libs which are not in the package.json, so we add it statically
+const COLLABORATIONS: AttributionLicense[] = [
+  {
+    license: 'http://www.apache.org/licenses/LICENSE-2.0',
```

Comment:
This is supposed to be human-readable text I think? So "Apache-2.0" would be more appropriate I think?